src/modules/auth/services/auth.service.ts

```diff
@@ -69,32 +69,36 @@ export class AuthService {
   async validateUser(email: string, password: string): Promise<UserDto> {
     const user = await this.userRepository.findOne({
       where: { email },
-      select: { password: true, is_verified: true },
+      select: { password: true, is_verified: true, email: true, id: true },
     });
 
     if (!user) {
       throw new NotFoundException(UsersServiceErrors.errors.NOT_FOUND(email));
     }
 
-    if (user && (await bcrypt.compare(password, user.password))) {
-      return user;
+    if (!(user && (await bcrypt.compare(password, user.password)))) {
+      throw new UnauthorizedException(
+        AuthServiceErrors.errors.INVALID_CREDENTIALS,
+      );
     }
-    throw new UnauthorizedException(
-      AuthServiceErrors.errors.INVALID_CREDENTIALS,
-    );
+
+    return user;
   }
 
   async login(user: UserDto): Promise<AccesResponseDto> {
```

Comment:
its service, why it accept full dto as input if use only part of data, service should input only data that will e used inside